privval/socket.go

```diff
@@ -18,11 +18,11 @@ import (
 )
 
 const (
-	defaultAcceptDeadlineSeconds = 3
-	defaultConnDeadlineSeconds   = 3
-	defaultConnHeartBeatSeconds  = 30
-	defaultConnWaitSeconds       = 60
-	defaultDialRetries           = 10
+	defaultAcceptDeadlineSeconds = 30 // tendermint waits this long for remote val to connect
+	defaultConnDeadlineSeconds   = 3  // must be set before each read
```

Comment:
`ConnReadDeadline`?

Reply:
Should we set it on writes too?

Reply:
Oh, if we have writes, then please ignore my comment. I haven't seen one...